cypress/e2e/patient_spec/patient_crud.cy.ts

```diff
@@ -2,9 +2,12 @@ import { cy, describe, before, beforeEach, it, afterEach } from "local-cypress";
 
 const username = "devdistrictadmin";
 const password = "Coronasafe@123";
-const phone_number = "9" + parseInt((Math.random() * 10 ** 9).toString());
-const emergency_phone_number = "9430123487";
-const yearOfBirth = "1999";
+const makePhoneNumber = () =>
+  "99" + Math.floor(Math.random() * 99999999).toString();
```

Comment:
Wouldn't this cause random fails?

Say the math.random generated 0.0000001 or some low values, wouldn't this result in a decrease number of digits?

Reply:
True, reverted the change.
Thanks!